source/manual/offsite-backup-and-restore.html.md

```diff
@@ -14,179 +14,184 @@ It hasn't been reviewed for accuracy yet.
 [View history in old opsmanual](https://github.com/alphagov/govuk-legacy-opsmanual/tree/master/2nd-line/offsite-backup-and-restore.md)
 
 
-We use [duplicity](http://duplicity.nongnu.org/) to perform offsite backups. Some
-backups are encrypted with GPG before being shipped to an Amazon S3 bucket.
+We use [duplicity](http://duplicity.nongnu.org/) to perform offsite backups. Some backups are encrypted with GPG before being shipped to an Amazon S3 bucket.
 
-The fingerprint of the key can be found in `hieradata/production.yaml` within
-the [govuk-puppet](https://github.com/alphagov/govuk-puppet) repository as
-`_: &offsite_gpg_key `.
+You will find the fingerprint of the key in `hieradata/production.yaml` within
+the [govuk-puppet](https://github.com/alphagov/govuk-puppet) repository.
 The key and passphrase are both stored in encrypted hieradata in the
-[deployment repository](https://github.digital.cabinet-office.gov.uk/gds/deployment),
-as `backup::assets::backup_private_gpg_key` and `backup::assets::backup_private_gpg_key_passphrase`
-respectively (the same private key is used for all offsite backups).
+[deployment repository](https://github.digital.cabinet-office.gov.uk/gds/deployment). The same private key is used for all offsite backups).
 
-## Rotating offsite backups GPG keys
-
-Please see
-<https://docs.publishing.service.gov.uk/manual/rotate-offsite-backup-gpg-keys.html>
+## Restore datastore from offsite backups
 
-## Restoring offsite backups of a datastore
+**Note**: Ensure you have followed the [Pre-requisites for restoring backups](#pre-requisites-for-restoring-backups) before attempting a backup restore.
 
-This requires access to 'production credentials hieradata' to retrieve the
-AWS credentials and GPG key to decrypt the backups.
+### Download a backup
 
-Connect to the machine where you want to restore the backup. For this example,
-we will try to restore and unpack a MySQL database on a Vagrant VM.
+1. Download the latest backup with:
 
-On a fresh VM, you may require the following packages for this exercise,
-installing using apt-get:
+      ```
+      duplicity restore --file-to-restore data/backups/whitehall-mysql-backup-1.backend.publishing.service.gov.uk/var/lib/automysqlbackup/latest.tbz2 s3://s3-eu-west-1.amazonaws.com/govuk-offsite-backups-production/govuk-datastores/ /tmp/latest.tbz2
+      ```
+  * When this completes you may see the following 'error':
 
-```
-sudo apt-get install duplicity python-pip python-boto mysql-server
-```
+        ```
```

Comment:
I've been going by how Middleman has been rendering https://govuk-tech-docs-pr-260.herokuapp.com/manual/offsite-backup-and-restore.html which looks ok.